backend/src/src-projects/src/main/java/org/eclipse/sw360/projects/ProjectHandler.java

```diff
@@ -279,6 +279,8 @@ public RequestStatus deleteProject(String id, User user) throws TException {
 
     @Override
     public List<Project> fillClearingStateSummary(List<Project> projects, User user) throws TException {
+        assertNotEmpty(projects, "Invalid project Ids!");
```

Comment:
Not sure, pls check whether assertion for empty project list is required and does not harm other endpoints calling the api,

![image](https://user-images.githubusercontent.com/56534710/125427369-74ea8a0f-ebb9-43ee-a714-b6cfdba7b062.png)

I guess , empty list is handled in `ProjectDatabaseHandler` - return empty list